.trivyignore

```diff
@@ -11,6 +11,8 @@ GHSA-f5pg-7wfw-84q9
 # these vulnerabilities require a malicious chart to be used.
 CVE-2024-25620
 CVE-2024-26147
+CVE-2025-32386
+CVE-2025-32387
```

Comment:
See the comment above which explains why these can be ignored for cert-manager.

Upgrading Helm module would have dragged in tons of non-patch updates to transitive dependencies